src/views/Layout.jsx

```diff
@@ -24,8 +24,8 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					{!!user ? <SignOutButton /> : <SignInButton />}
 					<div className="Nav-container">
+						{!!user ? <SignOutButton /> : <SignInButton />}
 						<button href="#" className="Nav-link">
 							Home
 						</button>
```

Comment:
So I know we will need to change these buttons to the NavLink comp from react-router-dom and I just not sure how the styling will work. 

I'm pretty sure it will be something like this but I know button elements have default styling so we may have to adjust the css once we make make sure the changing of the element works.